third_party/move/move-bytecode-verifier/transactional-tests/tests/signature/reference_as_type_actual_for_bytecode_instruction.exp

```diff
@@ -5,6 +5,6 @@ Error: Unable to publish module '00000000000000000000000000000000000000000000000
     major_status: INVALID_SIGNATURE_TOKEN,
     sub_status: None,
     location: 0x1::M,
-    indices: [(Signature, 2), (FunctionDefinition, 1)],
+    indices: [(FunctionInstantiation, 0)],
```

Comment:
Curious, why do we have all those baseline changes?

Reply:
It's because the new and old algorithms report error locations differently.
1. The new algorithm performs context-less checks, so it may not include a function definition index in the error location.
2. The old algorithm would include the signature index of the locals if there is anything wrong in a function definition, even though it may not have anything to do with those locals. The new algorithm gets this fixed.

Reply:
Was wondering if we should separate out the two tests: one for the old signature checker and one for the new one. You can pass in the config using arguments from the directive. I did sth similar for the paranoid mode testing for different outputs on when flag is enabled/disabled.

Reply:
Yeah I thought about something similar. Here are my thoughts
1. It would require some significant changes to the testing framework, in order to support two different implementations of the same bytecode verifier pass, unless you want to duplicate the existing tests
2. It would be a extra burden to maintain two sets of tests, and personally I don't see much value in doing that. The old code is not going to get updates or fixes. One should switch to the new implementation if they care about them.